frontend/src/modules/entitydetails/components/Metadata.css

```diff
@@ -49,3 +49,23 @@
     margin: 0;
     padding: 0;
 }
+
+.truncate {
+    text-overflow: ellipsis;
+    overflow: hidden;
+    padding-right: 9%;
```

Comment:
That's still not OK, see https://github.com/mozilla/pontoon/pull/1431#discussion_r339634630.

@adngdb How should we circumvent the problem of dynamic "See More" button width (due to localizations)? Should we dynamically calulcated it (and set padding in JS) or should we just use something like "..." and hardcode the width? I'm leaning towards latter.

Reply:
If we change the button text to "..." will it be clear that it is a button to expand even with it colored green?